src/wrappers/contract_wrappers/debt_token_wrapper.ts

```diff
@@ -560,7 +560,7 @@ export class DebtTokenContract extends BaseContract {
             return abiEncodedTransactionData;
         },
     };
-    public safeTransferFrom = {
+    public safeTransferFromWithData = {
```

Comment:
we deleted this out on the debt token api feature branch and just specified a default param for data.

Reply:
KK -- recreating.